_posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown

```diff
@@ -106,6 +108,36 @@ adopters and pod providers. We are very thankful for your patience and
 understanding while we work towards a smoothly functioning version 1 of
 CocoaPods and its architecture.
 
+### I got linked here after updating my gems, what to do?
+
+This issue is being tracked down to an issue located in Psych. To fix it please use the following command:
+
+```shell
+rm -rf "~/Library/Caches/CocoaPods ~/.cocoapods;" \
```

Comment:
@kylef @segiddins The commands are chained, one runs after the other is finished.

<img width="369" alt="screen shot 2015-12-09 at 21 35 35" src="https://cloud.githubusercontent.com/assets/1088217/11698004/de79e69c-9ebc-11e5-8fe8-83a63f330ef3.png">